### DISCUSSION TITLE
incorrect result while adding 4 numbers

### DISCUSSION THREAD

**Discussion by sitalsubedi777** [GorvGoyl/ChatGPT-Writer#128] [Report Bugs and Issues]
I told chat gpt to add (8700+11200+7200+5800), and the result was 33900, which is incorrect. I again request by stating its wrong but again shows the same result stating "It seems the original calculation was correct. If there's any discrepancy, please recheck the numbers or provide more information." I request the team to fix the issue.
![Chat gpt wrong answer](https://github.com/GorvGoyl/ChatGPT-Writer/assets/157869986/66f24c04-1bec-4d81-bc3c-3cbe9dc4a63f)

**Comment by sitalsubedi777** [comment]
However, it shows the correct an
![Chatgpt answer right](https://github.com/GorvGoyl/ChatGPT-Writer/assets/157869986/a5064402-505c-4948-8d13-9189a3df3c3d)
swer when I started it with 11200 followed by another number.

**Reply by abasrezaei** [reply]
من یک پروکسی چگونه ایجاد کنم
